Code/Editor/QtViewPaneManager.cpp

```diff
@@ -1144,6 +1144,25 @@ void QtViewPaneManager::RestoreDefaultLayout(bool resetSettings)
         m_mainWindow->addDockWidget(Qt::BottomDockWidgetArea, consoleViewPane->m_dockWidget);
         consoleViewPane->m_dockWidget->setFloating(false);
 
+        if (assetBrowserViewPane)
```

Comment:
General question for this part and the below logic that was removed: Do we want to keep the old default layout if `ed_useWIPAssetBrowserDesign` gets disabled by the user? If it looks OK in both scenarios, then we can leave the new layout for both cases.

Reply:
It looks okay in both scenarios, the old AB doesn't offer much functionality so it still looks and works fine when it is tabbed at the bottom with the console.